lib/aur-fetch

```diff
@@ -104,9 +104,13 @@ EOF
     exit 1
 }
 
-source /usr/share/makepkg/util/util.sh
```

Comment:
util.sh is still needed because `canonicalize_path` is used in the `-L` parseopt handling. L127